src/Http/Authentication.Core/src/Microsoft.AspNetCore.Authentication.Core.csproj

```diff
@@ -9,6 +9,8 @@
     <PackageTags>aspnetcore;authentication;security</PackageTags>
     <IsPackable>false</IsPackable>
     <Nullable>enable</Nullable>
+
+    <NoWarn />
```

Comment:
@dougbu \ @Pilchie how do you feel about this? Arcade makes 1591 a non-error: https://github.com/dotnet/arcade/blob/4873d157a8f34f8cc7e28b3f9938b32c642ef542/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectDefaults.props#L94 and this was the easiest way to suppress that behavior.

Reply:
Seems a big hammer in case there end up being other warnings we need to suppress.  An alternative is to add a `.editorconfig` file with:
```
[*.cs]
dotnet_diagnostic.CS1591.severity = error
```

In any directory where we are ready to enable this, which I think should be considered later than the `NoWarn`

Reply:
@wtgodbe suggested doing this instead: 

```xnk
<NoWarn>$(NoWarn.Replace('1591', ''))</NoWarn>
```

It is much more targeted. Our Directory.Build.props configures this error as a warning: https://github.com/dotnet/aspnetcore/blob/master/Directory.Build.props#L103. If we were to go with your suggestion I guess we would have to say

```
[*.cs]
dotnet_diagnostic.CS1591.severity = warning
```

Reply:
That works too.  I think as an end-goal in order to "stay-clean" we'd be better enabling the warning as an error once we don't have any violations.

Reply:
@scottaddie  who's the API ref resource

Reply:
> Our Directory.Build.props configures this error as a warning: https://github.com/dotnet/aspnetcore/blob/master/Directory.Build.props#L103

No, that line says the warning should not be treated as an error. Try with just your `$(NoWarn)` update in this project. Should work because project content is loaded after Directory.Build.props and all files it imports
``` msbuild
<NoWarn>$(NoWarn.Replace('1591', ''))</NoWarn>
```